### ISSUE TITLE
in fullsystem mode, Sv48 not support

### ISSUE THREAD

**Issue by SeriousAlpha** [OpenXiangShan/GEM5#204]
**Describe the bug**

- What workload are you running? Baremetal workload, bbl+Linux+application or checkpoints?
- What is the expected behavior? What is the actual behavior?
- Can this issue be consistently reproduced? If so, what are the steps to reproduce? Can this issue be reproduced in latest xs-dev branch?
- Can you upload your image or checkpoint to somewhere to help us to reproduce the issue?

**To Reproduce**
- Build command: typically `scons build/RISCV/gem5.opt`
- Running command: typically
```
mkdir -p util/xs_scripts/test
cd util/xs_scripts/test
bash ../kmh_6wide.sh /path/to/checkpoint
```
- The link to your image or checkpoint

**Expected behavior**

**Error log or Screenshots**

If applicable, plz provide you error log.

**Necessary information on versions**
- XS-GEM5 version: [e.g. xs-dev branch or commit: deadbeaf]
- NEMU version used as reference design: [e.g. master branch or commid: deadbeaf]
- Checkpoint restorer version used: [e.g. main branch of OpenXiangShan/LibCheckpointAlpha, or commit: deadbeaf of xxx]

**Additional information**
If you generate checkpoints your self, you can optionally provide version information of following components:
- riscv-linux
- riscv-rootfs
    - initramfs.txt
- riscv-pk or OpenSBI
- GCPT restorer (checkpoint restorer), it might be NEMU in older versions or OpenXiangShan/LibCheckpointAlpha or OpenXiangShan/LibCheckpoint in newer versions

**Additional context**

**Comment by eastonman** [comment]
Yes, there is just no SV48 code in the code base now. Not sure when SV48 will be added.

**Comment by jensen-yan** [comment]
sv48 模式现在已经支持，但可能仍有一些bug, 目前仍然推荐使用sv39 模式。
